packages/access-client/src/types.ts

```diff
@@ -60,6 +60,13 @@ export interface SpaceTableMetadata {
   agent: AgentMeta
 }
 
+/**
+ * Indicates failure executing ability that requires access to a space that cannot be found by the handler
+ */
+export type SpaceNotFoundFailure = Failure & {
+  status: 404
+}
+
```

Comment:
maybe create a new errors.ts file exported from types.ts with something like https://github.com/web3-storage/ucanto/blob/main/packages/interface/src/capability.ts#L413-L416

Reply:
I moved the failure type to an `errors.ts` file as you suggested https://github.com/web3-storage/w3protocol/pull/391/commits/8f0e5e342e858f25eb832bde4be2ea90a07175ae#diff-f4e938a2d743868395a4bb96adeecd10ce306ac53100e1e978df8d40ac33d4ef